cl/users/models.py

```diff
@@ -138,6 +138,10 @@ class UserProfile(models.Model):
         "subscribe to the case.",
         default=True,
     )
+    sort_desc = models.BooleanField(
```

Comment:
This isn't something that I feel really strongly, but I *think* this should probably mention dockets in the name, since CL is an ever-growing platform and `sort_desc` could refer to lots of things. Maybe a better name could be `desc_dockets` or `descending_dockets`, the latter would avoid the ambiguity of being interpreted as "description of dockets" or something like that?

Reply:
`docket_default_order_desc` ?

Reply:
Lengthy, but unambiguous. Sure.